test/spec/SearchResults/getRefinements.js

```diff
@@ -44,14 +41,12 @@ test('getRefinements(facetName) returns a refinement(facet) when a facet refinem
     attributeName: 'brand', count: 386, exhaustive: true, name: 'Apple', type: 'facet'
   }];
 
-  t.deepEqual(refinements, expected);
-  t.equal(refinements.length, refinedFacetValues.length);
-  t.ok(hasSameNames(refinements, refinedFacetValues));
-
-  t.end();
+  expect(refinements).toEqual(expected);
+  expect(refinements.length).toBe(refinedFacetValues.length);
+  expect(hasSameNames(refinements, refinedFacetValues)).toBeTruthy();
```

Comment:
We can make the assertion stricter.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453